src/Polymesh.ts

```diff
@@ -459,6 +460,40 @@ export class Polymesh {
     });
   }
 
+  /**
+   * Retrieve the event id of the token creation transaction
+   *
+   * @param args.ticker - Security Token ticker
+   */
+  public async getCreationTokenEventId(args: { ticker: string }): Promise<number | null> {
+    const {
+      context: { harvesterClient },
+    } = this;
+    const { ticker } = args;
+
+    let result: ApolloQueryResult<Pick<Query, 'eventByIndexedArgs'>>;
+    try {
+      result = await harvesterClient.query<Query>(
+        eventByIndexedArgs({
+          moduleId: 'asset',
+          eventId: 'AssetCreated',
+          eventArg1: ticker + '\u0000'.repeat(MAX_TICKER_LENGTH - ticker.length),
+        })
+      );
+    } catch (e) {
+      throw new PolymeshError({
+        code: ErrorCode.FatalError,
```

Comment:
I think we should have a specific error code for grapqhl errors. Something like `ErrorCode.MiddlewareError`